mlflow/store/dbfs_fuse_artifact_repo.py

```diff
@@ -0,0 +1,11 @@
+from mlflow.store.local_artifact_repo import LocalArtifactRepository
+
+
+class DbfsFuseArtifactRepository(LocalArtifactRepository):
```

Comment:
Do we need a dedicated class for this? Can't we just return local artifact repository directly?

Reply:
Yep we can just reuse `LocalArtifactRepository`, will change this